org.hedgewars.Hedgewars.yaml

```diff
@@ -0,0 +1,399 @@
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-utf8-string
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/utf8-string-1.0.1.1/utf8-string-1.0.1.1.tar.gz
+    sha256: fb0b9e3acbe0605bcd1c63e51f290a7bbbe6628dfa3294ff453e4235fbaef140
+  - type: shell
+    commands:
+    - sed -e 's/base >= 4.3 && < 4.9/base >= 4.3/' -i utf8-string.cabal
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-SHA
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/SHA-1.6.4.4/SHA-1.6.4.4.tar.gz
+    sha256: 6bd950df6b11a3998bb1452d875d2da043ee43385459afc5f16d471d25178b44
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-entropy
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/entropy-0.4.1.4/entropy-0.4.1.4.tar.gz
+    sha256: 2e3f6a65c8fde3551a8fb03b0a519b718762fc3278b1a5750f96d399e821eeb9
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-zlib
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/zlib-0.6.2/zlib-0.6.2.tar.gz
+    sha256: 0dcc7d925769bdbeb323f83b66884101084167501f11d74d21eb9bc515707fed
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-regex-base
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/regex-base-0.93.2/regex-base-0.93.2.tar.gz
+    sha256: 20dc5713a16f3d5e2e6d056b4beb9cfdc4368cd09fd56f47414c847705243278
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: haskell-regex-tdfa
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://hackage.haskell.org/package/regex-tdfa-1.2.3.1/regex-tdfa-1.2.3.1.tar.gz
+    sha256: 8aaaeeecf050807c7c514d4dd1763ac63bd121782de5a0847bef5d48a095ea50
+  build-commands:
+  - runhaskell Setup configure --prefix=/app
+  - runhaskell Setup build
+  - runhaskell Setup install
+  cleanup:
+  - '*'
+- name: hedgewars
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DBUILD_ENGINE_C=1
+  post-install:
+  - install -Dm0644 misc/hedgewars_ico.png /app/share/icons/hicolor/32x32/apps/hedgewars.png
+  - install -Dm0644 hedgewars_64.png /app/share/icons/hicolor/64x64/apps/hedgewars.png
+  - install -Dm0644 hedgewars_128.png /app/share/icons/hicolor/128x128/apps/hedgewars.png
+  - install -Dm0644 misc/hedgewars.png /app/share/icons/hicolor/512x512/apps/hedgewars.png
```

Comment:
You could build imagemagick to downscale the original icon, but it's a nitpick.

Reply:
Done